shift_allocation/tests/nh_clinical_staff_reallocation/test_reallocate.py

```diff
@@ -96,6 +96,9 @@ def mock_reallocation_write(*args, **kwargs):
                     allocatingwrite = args[4]
             return True
 
+        def mock_update_shift(*args, **kwargs):
+            pass
```

Comment:
Could we please raise tech debt to deal with this pass in the future, and comment the reason for this pass this time around? :) 
(This has already been discussed with Dan, logging the comment for traceability only)